src/solver.cpp

```diff
@@ -648,6 +649,9 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     Register(registry, ++id, Primitive::Mha, mha::Mha{}.SolverDbId());
     Register(registry, ++id, Primitive::Softmax, softmax::Softmax{}.SolverDbId());
     Register(registry, ++id, Primitive::Softmax, softmax::AttnSoftmax{}.SolverDbId());
+    Register(registry, ++id, Primitive::Loss, loss::HingeEmbeddingLossUnreducedFwd{}.SolverDbId());
+    Register(registry, ++id, Primitive::Loss, loss::HingeEmbeddingLossUnreducedBwd{}.SolverDbId());
+    Register(registry, ++id, Primitive::Loss, loss::HingeEmbeddingLossFwd{}.SolverDbId());
```

Comment:
Is there a reason HingeEmbeddingLossBwd is missing?

Reply:
It's my bad, i have added it.